code/modules/tables/presets.dm

```diff
@@ -52,6 +61,10 @@
 	table_mat = MATERIAL_GLASS
 	alpha = 77 // 0.3 * 255
 
+/obj/structure/table/glass/reinforced/marble
+	icon_state = "reinf_preview"
+
+
```

Comment:
```suggestion
```

Reply:
Derp.  Thanks!

Reply:
(Was an artifact of me trying out different table presets.)